go/teams/service_helper.go

```diff
@@ -1713,26 +1808,13 @@ func removeMultipleInviteIDs(ctx context.Context, team *Team, invIDs []keybase1.
 	return team.postTeamInvites(ctx, invites)
 }
 
-func removeInviteID(ctx context.Context, team *Team, invID keybase1.TeamInviteID, allowInaction bool) (err error) {
+func removeInviteID(ctx context.Context, team *Team, invID keybase1.TeamInviteID) (err error) {
 	defer team.MetaContext(ctx).Trace("remoteInviteID", &err)()
 	cancelList := []SCTeamInviteID{SCTeamInviteID(invID)}
 	invites := SCTeamInvites{
 		Cancel: &cancelList,
 	}
-	err = team.postTeamInvites(ctx, invites)
-	if allowInaction {
```

Comment:
what's with this? are we going to handle this differently?

Reply:
It's so if you fail to remove an invite, the rpc does nothing. People were double clicking the remove button. But in modern team mgmt (1) theres always a spinner and (2) it takes more than one click to remove someone, so I don't think this will be an issue

Reply:
kk thx